client/verta/setup.py

```diff
@@ -33,6 +33,7 @@
         "protobuf>=3.8",
         "pyyaml>=5.1, <5.4",
         "requests>=2.21, <3.0",
+        "numpy",
```

Comment:
Looks like we're going to need to add scipy and pandas as well, otherwise
```python
class Profilers:

    DEFAULT_ENVIRONMENT = Python(requirements=["numpy", "scipy", "pandas"])
````
fails at import time (for our linters) because `scipy` and `pandas` aren't present in the environment.